ui/components/src/OptionsEditorLayout/OptionsEditorGroup.tsx

```diff
@@ -12,31 +12,30 @@
 // limitations under the License.
 
 import { Box, Typography, Stack } from '@mui/material';
+import React from 'react';
 
 export type OptionsEditorGroupProps = {
   /**
    * Text that communicates the purpose of the grouping.
    */
   title: string;
   children: React.ReactNode;
+  icon?: React.ReactNode;
 };
 
 /**
  * Group similar content within panel options.
  */
-export const OptionsEditorGroup = ({ title, children }: OptionsEditorGroupProps) => {
+export const OptionsEditorGroup = ({ title, children, icon }: OptionsEditorGroupProps) => {
   return (
-    <Box>
-      <Typography
-        variant="overline"
-        component="h4"
-        borderBottom={1}
-        borderColor="grey.300"
-        marginBottom={(theme) => theme.spacing(1)}
-      >
-        {title}
-      </Typography>
+    <>
```

Comment:
Can we still wrap this in a `<Box>` to group the components together? It can cause unintended side-effects when using standalone similar to our [TimeRangeControls](https://github.com/perses/perses/pull/974).